core/console/src/test/java/org/eclipse/rdf4j/console/SetParametersTest.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Wrong header for new files. Should just read "Copyright (c) 2016 Eclipse RDF4J contributors."

Which reminds me that I need to update the code template to have the correct year.